app/controllers/api/account.php

```diff
@@ -33,6 +33,7 @@
 use Utopia\Validator\Assoc;
 use Utopia\Validator\Range;
 use Utopia\Validator\Text;
+use Utopia\Validator\JSON;
```

Comment:
Not sure this validator is used? I don't see it used in the changes.

Reply:
Leftover.